petab/v2/petab1to2.py

```diff
@@ -136,6 +136,9 @@ def _copy_file(src: Path | str, dest: Path | str):
     src = str(src)
     dest = str(dest)
 
+    if src == dest:
+        return
```

Comment:
Is this to skip computation or avoid an error? If the latter, then `src`/`dest` could be relative, so `Path.resolve()` should be used first.

Reply:
It's the latter. Agreed, resolve would be preferable. I didn't use it, because src/dest could be URLs. But on second thought, `shutil.copy` would break on those anyways...

Reply:
done